website/content/contributors/getting-started.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Do you want to start contributing to open source and do not know where to start?
+---
+
+### Introduction
+Open source is more than just a way of building software - it's a philosophy that promotes collaboration, transparency, and community. By sharing code openly, developers from all over the world can contribute their skills, knowledge, and creativity to projects that are used by millions of people every day. 
+
+The open source movement has changed the way we create, share, and innovate, and it continues to grow and evolve. Whether you're a seasoned programmer or just starting out, open source provides an incredible opportunity to learn new skills, work with like-minded people, and make a difference in the world. 
+
+In this document, we'll explore the different ways you can get involved in open source, what knowledge you need to contribute, and how you can become part of this amazing community. So, let's dive in and discover the exciting world of open source!
+
+### Join the Community
+To contribute to open source it's important to join the community. Most open source projects have a website or a [GitHub](https://github.com/) repository where you can find information on how to contribute. 
+
+Open source projects often have multiple communication channels to facilitate collaboration and support. The most common channels include mailing lists, forums, chat rooms, and issue trackers. In general, these communication channels are listed on the websites or the repositories of the projects. These channels allow project members and contributors to communicate effectively and efficiently, ask questions, share ideas, report issues, and propose solutions. When contributing to an open source project, it's important to use the appropriate communication channels and follow the community's guidelines for communication etiquette. Being an effective communicator is crucial to successful collaboration and contributing to the project's success.
```

Comment:
I think the CNCF falls on the side of "open source"

Here's a style guide from a while ago that touches on it: https://www.cncf.io/blog/2018/09/04/the-cloud-native-computing-foundation-cncf-style-guide/

Reply:
Thanks for this. @carolynvs the way I ready @nate-double-u 's style guide referance I agree with you.
Trust all is updated e.g. "open-source project", "open-source software", "open-source community"  and "Open source is more than just a way of building software", "To contribute to open source it's important"

Reply:
Unresolving because I think we should go through and use "open source" consistently, and without the hyphen.

See also https://github.com/cncf/foundation/blob/main/style-guide.md